spec/lib/binary-buffer-module.spec.js

```diff
@@ -0,0 +1,40 @@
+"use strict";
+
+describe("binaryBuffer", () => {
+    var binaryBuffer, copy;
+
+    beforeEach(() => {
+        binaryBuffer = require("../../lib/binary-buffer")();
+    });
+    [
+        "some string",
+        "",
+        new Buffer("some buffer"),
+        123
```

Comment:
This input is odd.  I don't mind it being there but it isn't a string and it isn't a buffer, so you're intentionally violating the input as described in the jsdoc for the method.